lib/rubocop/cop/chef/style/comments_format.rb

```diff
@@ -56,7 +56,8 @@ def investigate(processed_source)
         def autocorrect(comment)
           # Extract the type and the actual value. Strip out "Name" or "File"
           # 'Cookbook Name' should be 'Cookbook'. Also skip a :: if present
-          match = /^# ?([A-Za-z]+)\s?(?:Name|File)?(?:::)?\s(.*)/.match(comment.text)
+          # https://rubular.com/r/Do9fpLWXlCmvdJ
+          match = /^#\s*([A-Za-z]+)\s?(?:Name|File)?(?:::)?\s(.*)/.match(comment.text)
```

Comment:
This will also match `#Attributes File:: default` (no space) which I would think is desired

Reply:
Yep it'll match that as well.